es/provider.go

```diff
@@ -194,6 +195,12 @@ func Provider() *schema.Provider {
 				Default:     true,
 				Description: "Enable signing of AWS elasticsearch requests. The `url` must refer to AWS ES domain (`*.<region>.es.amazonaws.com`), or `aws_region` must be specified explicitly.",
 			},
+			"sign_aws_request_service_key": {
```

Comment:
What do you think about renaming this to `aws_signature_service` to have a mix between ease of understanding/short name? We should update the other references to be consistent as well.

Reply:
I thougt it might be helpful to group all Sig4 related variables using one prefix, that why I started with `sign_aws`. Seems like AWS calls this [service name](https://docs.aws.amazon.com/general/latest/gr/sigv4_elements.html), so I guess `aws_signature_service` is good 👍